### PR TITLE
Rn 2017 26

### PR DESCRIPTION
**Pull request by richardmateosian-okta**
## Description:
Produce release notes for 2017.26
## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* https://oktainc.atlassian.net/browse/OKTA-131641

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
@mystiberry-okta

### CODE REVIEW

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 1]
```diff
@@ -29,68 +29,43 @@ You can export data before Okta deletes it. We recommend using Security Informat
 
 ### Platform Enhancements
 
-* [System Logs Track Key Rotation and Generation](#system-logs-track-key-rotation-and-generation)
-* [Client Registration API Is an Early Access Feature](#client-registration-api-is-an-early-access-feature)
-* [Create OAuth 2.0 and OpenID Connect Clients with the Apps API](#create-oauth-20-and-openid-connect-clients-with-apps-api)
-* [OAuth 2.0 and OpenID Connect Client App Updates Available in System Log](#oauth-20-and-openid-connect-client-app-updates-available-in-system-log)
-* [Support for RP-Initiated Logout](#support-for-rp-initiated-logout)
-* [OAuth 2.0 and OpenID Connect .well-known Response Includes Registration Endpoint](#oauth-20-and-openid-connect-well-known-response-includes-registration-endpoint)
+* [Mask Client Secret](#mask-client-secret)
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: I was aiming for shorter titles this time. Perhaps I went overboard.

Comment (mystiberry-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: Up to five words is usually safe.

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 2]
```diff
@@ -29,68 +29,43 @@ You can export data before Okta deletes it. We recommend using Security Informat
...
+* [Support client_secret_jwt](#support-client_secret_jwt)
+* [Sign-In Widget](#sign-in-widget)
 
 
-#### System Logs Track Key Rotation and Generation
-Logged information about key rotation and generation for apps and identity providers is available by using GET requests to either of the following endpoints: `/api/v1/events` or `/api/v1/logs`.
-For more information, see [Identity Provider Signing Key Store Operations](https://developer.okta.com/docs/api/resources/idps.html#identity-provider-signing-key-store-operations)
-or [Update Key Credential for Application](https://developer.okta.com/docs/api/resources/apps.html#update-key-credential-for-application).
+#### Mask Client Secret
+When an Oauth 2.0 client fails multiple times within a specified period to log in successfully, the system log contains a masked representation of the client secret. The representation is always ten characters in length and accurately represents up to five initial characters of the secret.
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: OK. I didn't like it either.

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 3]
```diff
@@ -29,68 +29,43 @@ You can export data before Okta deletes it. We recommend using Security Informat
...
 
-* [System Logs Track Key Rotation and Generation](#system-logs-track-key-rotation-and-generation)
-* [Client Registration API Is an Early Access Feature](#client-registration-api-is-an-early-access-feature)
-* [Create OAuth 2.0 and OpenID Connect Clients with the Apps API](#create-oauth-20-and-openid-connect-clients-with-apps-api)
-* [OAuth 2.0 and OpenID Connect Client App Updates Available in System Log](#oauth-20-and-openid-connect-client-app-updates-available-in-system-log)
-* [Support for RP-Initiated Logout](#support-for-rp-initiated-logout)
-* [OAuth 2.0 and OpenID Connect .well-known Response Includes Registration Endpoint](#oauth-20-and-openid-connect-well-known-response-includes-registration-endpoint)
+* [Mask Client Secret](#mask-client-secret)
+* [Support client_secret_jwt](#support-client_secret_jwt)
+* [Sign-In Widget](#sign-in-widget)
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: OK

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 4]
```diff
@@ -29,68 +29,43 @@ You can export data before Okta deletes it. We recommend using Security Informat
...
+When an Oauth 2.0 client fails multiple times within a specified period to log in successfully, the system log contains a masked representation of the client secret. The representation is always ten characters in length and accurately represents up to five initial characters of the secret.
+The remaining characters are asterisks.
+{% img release_notes/MaskedClientSecret.png alt:"Masked Client Secret Event" %}
+<!-- (OKTA-129694) -->
 
-Here is a response from `/api/v1/logs`
-{% img release_notes/KeyRotateLog.png alt:"Logged Key Rotation Event" %}
-<!-- (OKTA-76607) -->
+#### Support client_secret_jwt
+Okta now supports the `client_secret_jwt` method for token endpoint authentication (`token_endpoint_auth_method`).
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: Happily!

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 5]
```diff
@@ -29,68 +29,43 @@ You can export data before Okta deletes it. We recommend using Security Informat
...
+This method is specified in the [OpenID Connect specification](http://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication), and allows you to use JWT and HMAC to authenticate a client for OAuth/OIDC requests.
+<!-- (OKTA-101074) -->
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: OK

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 6]
```diff
@@ -29,68 +29,43 @@ You can export data before Okta deletes it. We recommend using Security Informat
...
+* [Validate Scope Names](#validate-scope-names)
+* [Duplicate Create User Calls](#duplicate-create-user-calls)
 
 
-#### Invalid Availability of credentials.signing.kid
-The `credentials.signing.kid` property of an app was available even if its sign-on mode does not support
-certificates. Only apps using the following sign-on mode types support certificates: SAML 2.0, SAML 1.1,
-WS-Fed, or OpenID Connect. For more information,
-see: [Application Key Store Operations](https://developer.okta.com/docs/api/resources/apps.html#application-key-store-operations) (OKTA-76439)
+#### Validate Scope Names
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: Sorry, I did that last week too. Still rusty.

**Pull request merged**